woo-razorpay.php

```diff
@@ -1487,6 +1488,10 @@ public function process_refund($orderId, $amount = null, $reason = '')
                  */
                 do_action( 'woo_razorpay_refund_success', $refund->id, $orderId, $refund );
 
+                rzpLogInfo('Refund ID' . $refund->id);
```

Comment:
@yashgit891 , please have checks for object available before printing or using it.
Also make in array use all 3 logs print in one call.